primitives/io/src/lib.rs

```diff
@@ -1239,6 +1246,47 @@ pub trait Sandbox {
 	}
 }
 
+/// Runtime spawn extension.
+pub trait RuntimeSpawn : Send {
+	/// Create new runtime instance and use dynamic dispatch to invoke with specified payload.
+	///
+	/// Returns handle of the spawned task.
+	fn spawn_call(&self, dispatcher_ref: u32, func: u32, payload: Vec<u8>) -> u64;
```

Comment:
What's the deal with function pointer type? I was expecting `usize`, like a few other places.

Reply:
This is WASM pointer type, which is `u32`, I'll put a remark